src/components/title/edit/title-edit.js

```diff
@@ -124,6 +126,9 @@ class TitleEdit extends Component {
                 <EditionField />
                 <PublisherNameField />
                 <PublicationTypeField />
+                <ContributerField
```

Comment:
Definitely a nit, but we should probably fix that typo to prevent future copy-paste confusion. `Contributer` -> `Contributor`